src/backend/logconfig.py

```diff
@@ -22,6 +23,13 @@ def filter(self, record: logging.LogRecord) -> bool:
         return not record.name.startswith("watchfiles")
 
 
+def gzip_rotator(source, dest):
+    os.rename(source, dest)
```

Comment:
Why the rename?

Reply:
dunno. copy paste function nur leicht angepasst

Reply:
Ah, wahrscheinlich, dass nicht reingeschrieben wird während Gezirpes wird.